lib/changeNIDMtoSPM.m

```diff
@@ -22,11 +20,15 @@
     designMatrix = searchforType('nidm_DesignMatrix', graph);
     
     %Find the location of the design matrix.
-    locationID = searchforID(designMatrix{1}.('dc_description').('x_id'), graph);
-    nidmTemp.DesMat = getPathDetails(locationID.('prov_atLocation').('x_value'), json.filepath);
+    try
```

Comment:
Yes, the try/catch was replaced by an if/else as part of the changes submitted below.